samples/matter/thermostat/README.rst

```diff
@@ -157,27 +157,46 @@ Factory data support
 User interface
 **************
 
-.. include:: ../template/README.rst
-   :start-after: matter_template_nrf54l15_0_3_0_interface_start
-   :end-before: matter_template_nrf54l15_0_3_0_interface_end
+.. tabs::
 
-.. include:: ../lock/README.rst
-    :start-after: matter_door_lock_sample_led1_start
-    :end-before: matter_door_lock_sample_led1_end
+   .. group-tab:: nRF52, nRF53 and nRF70 DKs
 
-.. include:: ../lock/README.rst
-    :start-after: matter_door_lock_sample_button1_start
-    :end-before: matter_door_lock_sample_button1_end
+      LED 1:
+         .. include:: /includes/matter_sample_state_led.txt
 
-Button 2:
-    * Prints the most recent thermostat data to terminal.
+      Button 1:
+         .. include:: /includes/matter_sample_button.txt
 
-.. include:: ../lock/README.rst
-    :start-after: matter_door_lock_sample_jlink_start
-    :end-before: matter_door_lock_sample_jlink_end
+      Button 2:
+         Prints the most recent thermostat data to terminal.
+
+      .. include:: /includes/matter_segger_usb.txt
+
+      NFC port with antenna attached:
+          Optionally used for obtaining the `Onboarding information`_ from the Matter accessory device to start the :ref:`commissioning procedure <matter_thermostat_sample_remote_control>`.
+
+   .. group-tab:: nRF54 DKs
+
+      LED 0:
+         .. include:: /includes/matter_sample_state_led.txt
```

Comment:
There seems to be discrepancy between the tab with old numbering (it doesn't mention All LEDs, and the LED 2 that corresponds to the LED 1 mentioned in the nRF54 devices) - is this intended?

Reply:
Good catch. Seems to be a copy-paste error on my part, as that information is from the Light switch sample.